src/main/resources/www.liquibase.org/xml/ns/neo4j/neo4j-latest.xsd

```diff
@@ -0,0 +1,92 @@
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="http://www.liquibase.org/xml/ns/neo4j"
+            xmlns="http://www.liquibase.org/xml/ns/neo4j"
+            xmlns:core="http://www.liquibase.org/xml/ns/dbchangelog"
+            elementFormDefault="qualified">
+
+    <xsd:import namespace="https://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
I don't think you need to / should import the dbchangelog.xsd schema? I'm not **_exactly_** sure what that does, but your XSD should be stand-alone and separate from what gets defined in the other xsd. I think the import would make it so all the dbchangelog elements would also be available under the `neo4j:` namespace which I don't think we want.